lib/active_model/serializable_resource.rb

```diff
@@ -2,6 +2,15 @@
 module ActiveModel
   class SerializableResource
     ADAPTER_OPTION_KEYS = Set.new([:include, :fields, :adapter, :meta, :meta_key, :links])
+    extend ActiveModel::Callbacks
+
+    define_model_callbacks :render
+
+    around_render do |_, block, _|
+      notify_active_support do
+        block.call
+      end
+    end
```

Comment:
yeah, I think you're right. This doesn't add anything but complexity.  Sorry for having you extra work. I meant it as a thought if it made sense.. and it's more indirection than helpful